src/module_wrap.cc

```diff
@@ -607,8 +623,7 @@ void ModuleWrap::Evaluate(const FunctionCallbackInfo<Value>& args) {
   }
 
   if (try_catch.HasCaught()) {
-    if (!try_catch.HasTerminated())
-      try_catch.ReThrow();
+    if (!try_catch.HasTerminated()) try_catch.ReThrow();
```

Comment:
This looks like an unrelated change that's going to make backporting stuff needlessly harder, could we get rid of it?

Reply:
Fixed these up, thanks.